sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,272 @@
+
+* [Analyze text](#analyze-text)
+* [Analyze image](#analyze-image)
+* [Manage text blocklist](#manage-text-blocklist)
+
+### Analyze text
+
+#### Analyze text without blocklists
+```java com.azure.ai.contentsafety.analyzetext
+String endpoint = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_ENDPOINT");
+String key = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_KEY");
+ContentSafetyClient contentSafetyClient = new ContentSafetyClientBuilder()
+    .credential(new AzureKeyCredential(key))
+    .endpoint(endpoint).buildClient();
+
+AnalyzeTextResult response = contentSafetyClient.analyzeText(new AnalyzeTextOptions("This is text example"));
+
+System.out.println("Hate severity: " + response.getHateResult().getSeverity());
+System.out.println("SelfHarm severity: " + response.getSelfHarmResult().getSeverity());
+System.out.println("Sexual severity: " + response.getSexualResult().getSeverity());
+System.out.println("Violence severity: " + response.getViolenceResult().getSeverity());
+```
+
+#### Analyze text with blocklists
+```java com.azure.ai.contentsafety.analyzetextwithblocklist
+// After you edit your blocklist, it usually takes effect in 5 minutes, please wait some time before analyzing with blocklist after editing.
+AnalyzeTextOptions request = new AnalyzeTextOptions("I h*te you and I want to k*ll you");
+request.getBlocklistNames().add(blocklistName);
+request.setBreakByBlocklists(true);
+
+AnalyzeTextResult analyzeTextResult;
+try {
+    analyzeTextResult = contentSafetyClient.analyzeText(request);
+} catch (HttpResponseException ex) {
+    System.out.println("Analyze text failed.\nStatus code: " + ex.getResponse().getStatusCode() + ", Error message: " + ex.getMessage());
+    throw ex;
+}
+
+if (analyzeTextResult.getBlocklistsMatchResults() != null) {
+    System.out.println("\nBlocklist match result:");
+    for (TextBlocklistMatchResult matchResult : analyzeTextResult.getBlocklistsMatchResults()) {
+        System.out.println("Blockitem was hit in text: Offset: " + matchResult.getOffset() + ", Length: " + matchResult.getLength());
+        System.out.println("BlocklistName: " + matchResult.getBlocklistName() + ", BlockItemId: " + matchResult.getBlockItemId() + ", BlockItemText: " + matchResult.getBlockItemText());
+    }
+}
+```
+
+### Analyze image
+```java com.azure.ai.contentsafety.analyzeimage
+String endpoint = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_ENDPOINT");
+String key = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_KEY");
+
+ContentSafetyClient contentSafetyClient = new ContentSafetyClientBuilder()
+    .credential(new AzureKeyCredential(key))
+    .endpoint(endpoint).buildClient();
+
+ImageData image = new ImageData();
+String cwd = System.getProperty("user.dir");
+String absolutePath = cwd + "./src/image.jpg";
+image.setContent(Files.readAllBytes(Paths.get(absolutePath)));
```

Comment:
The method "readAllBytes()" can throw an IOException which is not handled. It is recommended to handle the exception.

Reply:
Added it into the function public void testAnalyzeImageTests() throws IOException
it is aligned with weidong before.